### PR TITLE
Clarify Documentation.

### PR DESCRIPTION
**Pull request by Carreau**
Writing Qt from within QtConsole is not different from any other REPL.
Saying that the QtConsole is special is even more confusing. which is
paradoxical as this section is trying to explain that QtConsole is not
different.

-- 
Prompted by a personal question I got by mail, in French.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -375,20 +375,39 @@ Rules:
...
+An important part of working with the REPL – QtConsole, Jupyter notebook,
+IPython terminal – when you are writing your own Qt code is to remember that
+user code (in the kernel) is *not* in the same process as the frontend.  This
+means that there is not necessarily any Qt code running in the kernel, and
+under most normal circumstances there isn't. This is true even if you are
+running the QtConsole.
+
+.. warning::
+
+    When executing code from teh qtconsole prompt, it is **not possible** to
```
Comment (takluyver) [inline docs/source/index.rst]: `teh` -> `the`

File: docs/source/index.rst [hunk 2]
```diff
@@ -375,20 +375,39 @@ Rules:
...
-Qt and the Qt console
-=====================
+Qt and the REPL
+===============
+
+.. note::
+
+    This section is relevant regardless of the frontend you use to write Qt
+    Code. This section is mostly there as it is easy to get confused and assume
+    that writing QtCode in the QtConsole should change from usual QtCode. It
```
Comment (takluyver) [inline docs/source/index.rst]: QtCode -> Qt code (x2)

File: docs/source/index.rst [hunk 3]
```diff
@@ -375,20 +375,39 @@ Rules:
...
+    should not. If you get confused, take a step back, and try writing your
+    code using the pure terminal based ``jupyter console`` that does not
+    involved Qt.
```
Comment (takluyver) [inline docs/source/index.rst]: involved -> involve

**Comment by Carreau** [comment]
fixed x4 .

**Pull request merged**